travis/script/build/build_frontend.sh

```diff
@@ -11,4 +11,5 @@ if [ -f "$ui_package_json" ]; then
   npm cache verify
   npm install
   npm run build
+  npm run test
```

Comment:
It seems like the test command does not print test results to stdout in Travis.

Reply:
It would be nice to show the test results in the console output. We are currently using Travis without an external storage service. Therefore all important information are written to the console. The xml is removed after the Travis build.

Reply:
Alright. Revert it is.